staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -269,14 +269,17 @@ func GetHistogramVecFromGatherer(gatherer metrics.Gatherer, metricName string, l
 		return nil, fmt.Errorf("metric %q not found", metricName)
 	}
 
-	if metricFamily.GetMetric() == nil {
-		return nil, fmt.Errorf("metric %q is empty", metricName)
-	}
-
 	if len(metricFamily.GetMetric()) == 0 {
 		return nil, fmt.Errorf("metric %q is empty", metricName)
 	}
 
+	return metricFamily, nil
+}
+
+// GetHistogramVec collects a vector of Histogram from a metric family, that matches the input labelValue map.
+// If the labelValue is nil, all data of metric will be aggregated and returned, otherwise metric with the labels that match
+// with labelValue map will be aggregated accordingly.
```

Comment:
I think this comment should be applied to the caller, right? as this function itself is agnostic to how the returned data gets used.

And I noticed you already put comments in the caller side (`collect()`), so just remove these 2 lines should be good.

Reply:
Done